apps/api/v2/src/modules/organizations/controllers/outputs/listUsersResponse.dto.ts

```diff
@@ -0,0 +1,13 @@
+import { IsArray } from "class-validator";
+
+export interface User {
+  id: number;
+  name: string | null;
+  email: string;
+  timeZone: string;
+}
+
+export class ListUsersResponseDto {
```

Comment:
I think we don't need `List` in name since having `Users` in plural already would imply returning some kind of an array.